OpenSim/Simulation/SimbodyEngine/Joint.cpp

```diff
@@ -152,6 +152,17 @@ Joint::CoordinateIndex Joint::constructCoordinate(Coordinate::MotionType mt)
     return cix;
 }
 
+Joint::CoordinateIndex Joint::constructCoordinate(Coordinate::MotionType mt,
+                                                  unsigned idx)
+{
+    const auto cix = constructCoordinate(mt);
```

Comment:
Are you keeping the previous incarnation of `constructCoordinate` so that all the other Joints remain functional while you test this change? Are you then planning to have just one implementation of `constructCoordinate` after updating the other Joint types? I think the incremental changes are good but you should open an accompanying issue for what changes are forthcoming. You can also make `constructCoordinate(mt)` private to identify all the Joints that fail. Leaving it in as is risks having some Joints still on the "old" system without enumerators that we would not know about, or new Joints being implemented in the old way.